packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -515,6 +523,26 @@ const helperBehavior = {
         facets.manager.burnAndRecord(accounting.toBurn, vaultSeat);
 
         // current values
+
+        // Sometimes, the AMM will sell less than all the collateral. If there
+        // was a shortfall, the investor doesn't keep the change, so we get it
```

Comment:
just FYI: this doesn't end in a period while the other comment block does.

Reply:
done